README.md

```diff
@@ -74,7 +74,9 @@ Initializing the SDK requires you to provide two values, an Appcues account ID,
 
 #### Supporting Debugging and Experience Previewing
 
-Supporting debugging and experience previewing is not required for the Appcues React Native Module to function, but it is necessary for the optimal Appcues builder experience. Refer to the [URL Scheme Configuration Guide](https://github.com/appcues/appcues-react-native-module/blob/main/docs/URLSchemeConfiguring.md) for details on how to configure.
+During installation, follow the steps outlined in [Configuring the Appcues URL Scheme](https://github.com/appcues/appcues-react-native-module/blob/main/docs/URLSchemeConfiguring.md). 
+
+> [!IMPORTANT] This is necessary for the optimal Appcues builder experience, to support experience preview, screen capture and debugging.
```

Comment:
I think you need a newline after `[!IMPORTANT]`? https://blog.jakelee.co.uk/github-alert-experiments/#syntax

Reply:
And maybe instead of "optimal" say "complete"? It's pretty much required, especially with screen capture/anchored tooltips.

Reply:
yep, weird, I force pushed an update to this branch but the PR keeps showing me the older commit here for some reason.

Reply:
ok, now its updated.

![Screenshot 2023-07-31 at 11 56 16 AM](https://github.com/appcues/appcues-react-native-module/assets/19266448/803f4787-ef23-4a7c-83b1-41591da81f4c)